package.json

```diff
@@ -159,6 +159,16 @@
         "command": "quarkusTools.deployToOpenShift.short",
         "title": "Deploy to OpenShift",
         "icon": "icons/openshift.png"
+      },
+      {
+        "command": "qute.validation.enabled.toggle.on",
```

Comment:
So when you open a file, and validation is on, the command you run is toggle on ? Maybe it should be renamed to toggle off, so it makes more sense. It's really is to get confused on what something does when you redefine terminology like this.

Reply:
I did it by naming convention since there are 2 separate buttons; if the open eye is shown, and the button is hovered, it displays that Qute validation is enabled for the current file. The inverse effect happens when clicking it (which is confusing implementation wise) where if you click the toggle on button, the button will switch to the eye closed button, but the toggle off effect will occur. The reverse happens for the toggle off state. I'm open to changing this however.

Reply:
I just saw that you called the function `toggle.on`, and in it, you set `quteValidationEnabled` to `true`. Maybe if the function would have been called `toggle.off` it would have immediately made sense what the result should be.

Reply:
Should the title be changed as well?

Reply:
I would say yes because you're using the title to indicate the current state, but that should (hopefully) be achieved by the icon. It's common for the icon's hover text (title) to indicate what happens when you press it (eg. Enable Qute Validation)